pdns/rec_control.cc

```diff
@@ -96,7 +96,8 @@ int main(int argc, char** argv)
     "dump-rpz",
     "dump-throttlemap",
     "dump-non-resolving",
-    "dump-saved-parent-ns-sets"};
+    "dump-saved-parent-ns-sets",
+    "dump-dot-probe-map"};
```

Comment:
Don't suppose there would be a way to have a trailing comma (maybe a "" entry?) so that blame wouldn't need to keep shifting.